### PR TITLE
feat: redesign join conversation page

### PR DESCRIPTION
**Pull request by aweiss-dev**
Task: https://wearezeta.atlassian.net/browse/WPB-3789

Redesign: 
redesign
Updated:
Screenshot 2023-08-18 at 16 51 15

### CODE REVIEW

File: src/script/auth/page/ConversationJoin.tsx [hunk 1]
```diff
@@ -21,64 +21,92 @@ import React, {useEffect, useState} from 'react';
...
+  return (
+    <div
+      style={{
+        display: 'flex',
+        alignItems: 'center',
+        maxWidth: `${!isMobile ? '4rem' : '100%'}`,
+        justifyContent: 'center',
+        transform: `rotate(${!isMobile ? '90' : '0'}deg)`,
+        marginLeft: `${!isMobile ? '' : '16px'}`,
+      }}
```
Comment (atomrc) [inline src/script/auth/page/ConversationJoin.tsx]: let's extract that to a `style` file and put it in the `css` property (not `style`)

File: src/script/auth/page/ConversationJoin.tsx [hunk 2]
```diff
@@ -21,64 +21,92 @@ import React, {useEffect, useState} from 'react';
...
+const Separator = () => {
+  const isMobile = useMatchMedia(QUERY.mobile);
+  const Line = () => (
+    <div
+      style={{
+        flex: 1,
+        height: '1px',
+        backgroundColor: '#696c6e',
+        minWidth: '24rem',
+      }}
```
Comment (atomrc) [inline src/script/auth/page/ConversationJoin.tsx]: let's extract that to a style file and put it in the css property (not style)

Comment (tlebon) [reply src/script/auth/page/ConversationJoin.tsx]: probably a question i will regret asking:
the rest of the file has all inline styling. it doesnt make sense to just move one style to a separate file.

Comment (tlebon) [reply src/script/auth/page/ConversationJoin.tsx]: shall i leave all of them inline, or change all of them to a separate file 😨

File: src/script/auth/page/ConversationJoin.tsx [hunk 3]
```diff
@@ -201,132 +231,149 @@ const ConversationJoinComponent = ({
...
-                      data-uie-name="enter-name"
-                    />
-                    <RoundIconButton
-                      disabled={!enteredName || !isValidName || isSubmitingName}
-                      type="submit"
-                      formNoValidate
-                      onClick={checkNameValidity}
-                      data-uie-name="do-next"
+        <AppAlreadyOpen />
+        <div style={{display: 'flex', alignItems: 'center', flexDirection: 'column', marginBottom: '2rem'}}>
```
Comment (atomrc) [inline src/script/auth/page/ConversationJoin.tsx:211]: would probably be beneficial to split that huge jsx part into mutliple smaller components

**Review by atomrc — APPROVED** [review]

File: src/script/auth/page/ConversationJoin.tsx (hunk 2 above)
Comment (tlebon) [reply src/script/auth/page/ConversationJoin.tsx]: for some reason `css` is being parsed differently than `style`, breaking the page. as such i am leaving it as is for now.

**Pull request merged**